api/XMLHttpRequest.json

```diff
@@ -1635,7 +1635,7 @@
           "spec_url": "https://xhr.spec.whatwg.org/#the-upload-attribute",
           "support": {
             "chrome": {
-              "version_added": "1"
+              "version_added": "2"
```

Comment:
Safari says 4, so that's consistent with support not existing in WebKit long before Chrome.